meinberlin/apps/budgeting/assets/PaginationButton.jsx

```diff
@@ -16,6 +16,8 @@ export const PaginationButton = (props) => {
       return <i className="fa fa-chevron-left" aria-hidden="true" />
     } else if (props.type === 'next') {
       return <i className="fa fa-chevron-right" aria-hidden="true" />
+    } else if (props.type === 'mobile') {
+      return `${props.pageIndex} / ${props.pageCount}`
```

Comment:
the text color is grey for this and not very readable, I assume we should also use the default black here?